corpus/impulse/views.py

```diff
@@ -492,7 +492,12 @@ def mark_payment_complete(request, pk):
             send_email(
                 "Payment Complete | Impulse",
                 "emails/impulse/payment_complete.html",
-                {"team": team, "user": member.user, "payment_status": payment_status, "final_message": final_message},
+                {
+                    "team": team,
+                    "user": member.user,
+                    "payment_status": payment_status,
+                    "final_message": final_message,
+                },
```

Comment:
Hopefully all these changes were made by the linter?

Reply:
Yes, I did not touch these